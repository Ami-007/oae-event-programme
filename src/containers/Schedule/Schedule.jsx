import React from "react";
import ScheduleItem from "../../components/ScheduleItem/ScheduleItem"

const Schedule = () => {


    return (
        <div className="schedule">
            <ScheduleItem header="Welome" isImg="false" />
            <ScheduleItem header="header" subHeader="subheader" description="lorem ipsum whatever"/>
            <ScheduleItem header="header" subHeader="subheader" description="lorem ipsum whatever"/>
        </div>
    )
}

export default Schedule;