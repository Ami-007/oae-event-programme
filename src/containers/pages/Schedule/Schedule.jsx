import React from 'react';
import { Link } from 'react-router-dom';

const Schedule = () => {
  return (
    <div>

      <h1>Schedule page</h1>
    </div>
  )
}

export default Schedule
