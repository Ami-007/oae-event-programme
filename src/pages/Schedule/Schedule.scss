.schedule {
  padding: 2rem;
}
