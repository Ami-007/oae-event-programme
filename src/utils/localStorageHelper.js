export const getDarkTheme = () => {
  //   if (darkTheme == true) {
  //     localStorage.setItem("--background-color", darkTheme);
  //   } else {
  //     localStorage.setItem("theme", "light");
  //   }
  console.log(localStorage.getItem("darkMode"));

  // convert isDarkMode to a Boolean
  // return that boolean

  return true;
};

export const setDarkTheme = (isDarkMode) => {
  localStorage.setItem("darkMode", isDarkMode.toString());
};
