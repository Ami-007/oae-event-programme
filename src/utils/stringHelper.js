export const getFontType = (fontType) => {
  if (typeof fontType === "string") {
    fontType = parseInt(fontType);
  }
  if (fontType === 1) {
    return "modern";
  } else if (fontType === 2) {
    return "classic";
  } else {
    return "modern";
  }
};

export default getFontType;
