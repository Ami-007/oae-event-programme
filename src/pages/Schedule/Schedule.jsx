<<<<<<< HEAD
import React, { Fragment } from "react";
=======
import React from "react";
import { useParams } from "react-router-dom";
import Layout from "../../components/Layout/Layout";
>>>>>>> 121cae44a7869ba7d1d28da4535824af62951851
import ScheduleItem from "../../components/ScheduleItem/ScheduleItem";
import "./Schedule.scss";
import PageHeader from "../../components/PageHeader/PageHeader";

const Schedule = () => {
<<<<<<< HEAD
  return (    
    <Fragment>
    <PageHeader title="can you hear the shape of the drums?" subtitle="Event Schedule." />
    <div className="schedule">      
      <ScheduleItem header="Welcome" />
      <ScheduleItem
        header="header"
        subHeader="subheader"
        description="lorem ipsum"
      />
      <ScheduleItem
        header="header"
        subHeader="subheader"
        description="lorem ipsum"
      />
    </div>
    </Fragment>
=======
  const { eventId } = useParams();

  return (
    <Layout eventId={eventId}>
      <div className="schedule">
        <ScheduleItem header="Welcome" />
        <ScheduleItem
          header="header"
          subHeader="subheader"
          description="lorem ipsum"
        />
        <ScheduleItem
          header="header"
          subHeader="subheader"
          description="lorem ipsum"
        />
      </div>
    </Layout>
>>>>>>> 121cae44a7869ba7d1d28da4535824af62951851
  );
};

export default Schedule;
