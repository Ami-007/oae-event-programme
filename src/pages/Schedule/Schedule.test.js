import { customRender } from "../../utils/testUtils";
import Schedule from "./Schedule";

jest.mock("react-router-dom", () => ({
  ...jest.requireActual("react-router-dom"),
  useParams: () => ({
    eventId: "6bb8ffe8-6e5e-4abc-be7f-b05b749849cf",
  }),
}));

test("Test", () => {
  const { container } = customRender(<Schedule />);
  expect(container).toMatchSnapshot();
});
