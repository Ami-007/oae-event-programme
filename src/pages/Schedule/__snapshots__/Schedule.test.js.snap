// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test 1`] = `
<div>
  <div
    class="layout modern"
  >
    <div>
      <div
        class="page-header page-header--only"
      >
        <div
          class="header-home"
        >
          <span
            class="header-home__subtitle"
          >
            BACH, THE UNIVERSE & EVERYTHING
          </span>
          <h1
            class="header-home__title"
          >
            Event Schedule.
          </h1>
        </div>
      </div>
      <div
        class="schedule"
      >
        <div
          class="schedule-item"
        >
          <div
            class="schedule-item__top"
          >
            <span
              class="schedule-item__headers"
            >
              <h3
                class="schedule-item__header"
              >
                Welcome
              </h3>
              <p
                class="schedule-item__subheader"
              />
            </span>
            <span
              class="schedule-item__button-container"
            >
              <img
                alt=""
                class="schedule-item__button"
                role="button"
                src=""
              />
            </span>
          </div>
          <hr
            class="schedule-item__hr"
          />
        </div>
        <div
          class="schedule-item"
        >
          <div
            class="schedule-item__top"
          >
            <span
              class="schedule-item__headers"
            >
              <h3
                class="schedule-item__header"
              >
                Sunt est irure occaecat qui duis enim adipisicing sint nisi ullamco nisi enim ullamco dolore.
              </h3>
              <p
                class="schedule-item__subheader"
              />
            </span>
            <span
              class="schedule-item__button-container"
            >
              <img
                alt=""
                class="schedule-item__button"
                role="button"
                src="chevron-down.png"
              />
            </span>
          </div>
          <hr
            class="schedule-item__hr"
          />
        </div>
        <div
          class="schedule-item"
        >
          <div
            class="schedule-item__top"
          >
            <span
              class="schedule-item__headers"
            >
              <h3
                class="schedule-item__header"
              >
                Cantata
              </h3>
              <p
                class="schedule-item__subheader"
              >
                Ach wie fluchtig, ach wie nichtig BWV 26
              </p>
            </span>
            <span
              class="schedule-item__button-container"
            >
              <img
                alt=""
                class="schedule-item__button"
                role="button"
                src="chevron-down.png"
              />
            </span>
          </div>
          <hr
            class="schedule-item__hr"
          />
        </div>
      </div>
    </div>
    <nav
      class="navbar"
    >
      <div
        class="navbar__content"
      >
        <a
          class="navbar__schedule-link"
          href="/6bb8ffe8-6e5e-4abc-be7f-b05b749849cf/event-schedule"
        >
          <img
            alt="schedule icon"
            class="navbar__icon"
            src="schedule.svg"
          />
        </a>
        <span
          class="navbar__title"
        >
          SCHEDULE
        </span>
      </div>
      <div
        class="navbar__content"
      >
        <a
          class="navbar__home-link"
          href="/6bb8ffe8-6e5e-4abc-be7f-b05b749849cf"
        >
          <img
            alt="home icon"
            class="navbar__icon"
            src="home.svg"
          />
        </a>
        <span
          class="navbar__title"
        >
          HOME
        </span>
      </div>
      <div
        class="navbar__content"
      >
        <a
          class="navbar__about-link"
          href="/6bb8ffe8-6e5e-4abc-be7f-b05b749849cf/our-story"
        >
          <img
            alt="about icon"
            class="navbar__icon"
            src="info.svg"
          />
        </a>
        <span
          class="navbar__title"
        >
          ABOUT
        </span>
      </div>
    </nav>
  </div>
</div>
`;
